modules/doc/dataloop.json

```diff
@@ -0,0 +1,122 @@
+{
+  "displayName": "DOC to TXT",
+  "name": "doc_to_txt",
+  "scope": "public",
+  "version": "0.0.37",
+  "description": "An application for converting DOC to TXT items",
+  "attributes": {
+    "Category": "Application",
+    "Provider": "Dataloop",
+    "Deployed By": "Dataloop",
+    "Application Type": [
+      "Pipeline Node"
+    ],
+    "Media Type": [
+      "Text"
+    ]
+  },
+  "codebase": {
+    "type": "git",
+    "gitUrl": "https://github.com/dataloop-ai-apps/document-preprocessing.git",
+    "gitTag": "0.0.37"
+  },
+  "components": {
+    "computeConfigs": [
+      {
+        "name": "doc-to-txt-v2",
+        "runtime": {
+          "podType": "regular-m",
+          "runnerImage": "gcr.io/viewo-g/piper/agent/runner/apps/document-preprocessing:0.1.2",
+          "concurrency": 10,
+          "autoscaler": {
+            "minReplicas": 0,
+            "maxReplicas": 2,
+            "queueLength": 10
+          }
+        }
+      }
+    ],
+    "pipelineNodes": [
+      {
+        "invoke": {
+          "type": "function",
+          "namespace": "doc-to-txt-v2.doc_to_txt_v2.doc_extraction"
+        },
+        "name": "doc_to_txt",
+        "categories": [
+          "Text-Preprocessing"
+        ],
+        "displayName": "DOC to TXT",
+        "description": "Converting DOC to Txt items",
+        "scope": "node",
```

Comment:
Scope project?

Reply:
it will effect on the service so the service will be by project and not by node.. this is how we want it?